$OpenBSD$

attempt to fix
Need an operator in 'autodeps/Linux/src/printmanagerstream.dep'

Index: Code/make_lib.mk
--- Code/make_lib.mk.orig
+++ Code/make_lib.mk
@@ -27,5 +27,6 @@ clean:
 	rm -rf $(BUILDS) $(DEPDIR)
 
 ### automatic dependency generation
-$(DEP_PATH)/%.dep: %.cpp; @mkdir -p $(shell dirname $@) && $(MAKEDEPEND)
+$(DEP_PATH)/%.dep: %.cpp
+@mkdir -p $(shell dirname $@) && $(MAKEDEPEND)
 -include $(CXXFILES:%.cpp=$(DEP_PATH)/%.dep)
